scripts/scil_tractogram_cut_streamlines.py

```diff
@@ -126,32 +132,36 @@ def main():
     if len(sft.streamlines) == 0:
         parser.error('Input tractogram is empty.')
 
-    # Mask scenario. Streamlines outside of the mask will be cut.
-    if args.mask and args.keep_longest:
-        mask_img = nib.load(args.mask)
-        binary_mask = get_data_as_mask(mask_img)
-
-        new_sft = cut_streamlines_with_mask(
-            sft, binary_mask, cutting_style=CuttingStyle.KEEP_LONGEST,
-            min_len=args.min_length, processes=args.nbr_processes)
-    # Mask scenario, only trimming the endpoints of the streamlines outside of
-    # the mask.
-    elif args.mask and args.trim_endpoints:
-        mask_img = nib.load(args.mask)
-        binary_mask = get_data_as_mask(mask_img)
-
-        new_sft = cut_streamlines_with_mask(
-            sft, binary_mask, cutting_style=CuttingStyle.TRIM_ENDPOINTS,
-            min_len=args.min_length, processes=args.nbr_processes)
-    # Mask scenario, keeping the longest segment of the streamline that is in
-    # the mask.
-    elif args.mask:
-        mask_img = nib.load(args.mask)
-        binary_mask = get_data_as_mask(mask_img)
-
-        new_sft = cut_streamlines_with_mask(
-            sft, binary_mask, cutting_style=CuttingStyle.DEFAULT,
-            min_len=args.min_length, processes=args.nbr_processes)
+    if args.mask:
+        style = args_to_style[args.keep_longest, args.trim_endpoints]
+        # Mask scenario, keeping the longest segment of the streamline that is
+        # in the mask.
+        if style == CuttingStyle.KEEP_LONGEST:
```

Comment:
Nice addition of the args_to_style function here. If I am not mistaken, you can now simply remove all the ifs about style and have the whole block only once (from mask_img to new_sft), since the variable style takes care of the changes.